src/game/platform_displacement.c

```diff
@@ -69,6 +69,9 @@ void update_mario_platform(void) {
 void update_platform_displacement_info(struct PlatformDisplacementInfo *displaceInfo, Vec3f pos, s16 yaw, struct Object *platform) {
     Vec3f scaledPos, yawVec, localPos;
 
+    // Avoid a crash if the platform unloaded its collision while stood on or is static
+    if (platform->header.gfx.throwMatrix == NULL) return;
```

Comment:
Ehh...I'm not a fan of inlining returns like this unless they're part of condensed switch blocks. It makes them easy to miss even if they're here as a safeguard.

Reply:
this is how it already was in the other function the entire time...

Reply:
This may be true and I probably just missed it before but I still don't like it lol